source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml

```diff
@@ -0,0 +1,21 @@
+﻿<metadata>
+    <attr 
+        path="/api/package[@name='com.google.android.gms.tflite.internal.tflite_gpu']" 
+        name="managedName"
+        >
+        Android.Gms.TfLite.Internal.TfLiteGpu
+    </attr>
+    <attr 
+        path="/api/package[@name='com.google.android.gms.tflite.gpu']" 
+        name="managedName"
+        >
+        Android.Gms.TfLite.TfLite.Gpu
```

Comment:
Let's remove the duplicate `TfLite` here.

Reply:
copypasta mistake